selector/__init__.py

```diff
@@ -470,7 +468,7 @@ def select(self, visits: List[Visit],
                         wind_conditions = actual_wind_conditions.get_wind_conditions(obs.visibility.azimuth[inight])
                     
                     # NOTE: first condition always going to be true. Why is needed?
-                    if too_status != 'rapid' and obs.too_status != 'none':
+                    if too_status != ToOType.RAPID and obs.too_status != ToOType.NONE:
```

Comment:
There is a need for `ToOType.None` instead of just using `None`?

Reply:
I think it's just a matter of semantics. If `too_status` is `None`, that means that it could not be read.
If it is `ToOType.NONE`, that means that it was read correctly and was deliberately specified as `ToOType.NONE`, so there is a subtle distinction.